pkg/storage/batch.go

```diff
@@ -166,7 +166,7 @@ func (it *batchChunkIterator) nextBatch() (genericIterator, error) {
 			batch = append(batch, it.lastOverlapping...)
 		}
 
-		includesOverlap = true
+		//includesOverlap = true
```

Comment:
Is this a mistake?

Reply:
it is! good catch, commented this out at one point to see if it was the issue, lemme fix.